examples/react-hook-form/pages/url-form/[index].tsx

```diff
@@ -0,0 +1,263 @@
+import { memo, useEffect, useRef } from 'react'
+import type { GetServerSideProps, NextPage } from 'next'
+import Head from 'next/head'
+import Link from 'next/link'
+import Router from 'next/router'
+import { SubmitHandler } from 'react-hook-form/dist/types/form'
+import { syncEffect } from 'recoil-sync'
+import { array, object, string } from '@recoiljs/refine'
+import {
+  initializableAtomFamily,
+  RecoilURLSyncJSONNext,
```

Comment:
nits: It can be deleted.
```suggestion
import {
  initializableAtomFamily,
```

Reply:
Thanks, fixed.